models/multi_organ_segmentation/configs/metadata.json

```diff
@@ -0,0 +1,90 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "version": "0.0.1",
+    "changelog": {
+        "0.0.1": "initialize the model package structure"
+    },
+    "monai_version": "1.2.0rc6",
+    "pytorch_version": "1.13.0",
+    "numpy_version": "1.21.2",
+    "optional_packages_version": {
+        "fire": "0.4.0",
+        "nibabel": "4.0.1",
+        "pytorch-ignite": "0.4.9"
+    },
```

Comment:
Hi @holgerroth @cshen0625 , we decided to add a new key "name" in metadata after my last review.
It is used as a display name of the bundle in the webpage.
Could you please help to add it? Thanks!

Reply:
@yiheng-wang-nv Please provide an example in some bundle for reference.

Thanks.

Reply:
here is an example:
https://github.com/Project-MONAI/model-zoo/blob/b72b9fbe80e999407d4d136207778a02f37d3c6d/models/brats_mri_axial_slices_generative_diffusion/configs/metadata.json#L18

Reply:
Hi @cshen0625 , could you please also address this?
We added the requirement of the "name" (see: https://github.com/Project-MONAI/model-zoo/blob/b245833e22544126f23925eac89da471562467a1/CONTRIBUTING.md#model-naming)